lib/vagrant-openshift/constants.rb

```diff
@@ -47,12 +51,19 @@ def self.aggregated_logging_repos
         }
       end
 
+       def self.metrics_repos
```

Comment:
I'm not sure about this pattern. Why is this method defined twice? The first definition just calls this one. I'm not sure this works the way you might be expecting it to. Maybe you could get rid of the first definition and change this one to something like
`def self.metrics_repos(env=nil)`
?